lib/editor/components/EntityListButtons.js

```diff
@@ -169,7 +169,9 @@ export default class EntityListButtons extends Component<Props, State> {
             bsSize='small'
             data-test-id={`new-${activeComponent}-button`}
             disabled={entities && entities.findIndex(entityIsNew) !== -1}
-            onClick={this._onClickNew}>
+            onClick={this._onClickNew}
+            // Setting the max width to display value so the button does not move to new line
```

Comment:
Add "based on the width of the 'New calendar' button".

Reply:
Add also a note that the width of the button text varies depending on the font used.